scripts/check-test-determinism.py

```
@@ -626,6 +1185,99 @@ def _self_test() -> int:
+        ),
+        (
+            "web/tests/spawn_shell_argv.ts",
+            'spawn("sh", ["-c", "curl https://api.openai.com/v1/items"])\n',
+            {RULE_LIVE_NETWORK_HOST},
+        ),
+        (
+            "tests/multiline_subprocess_shell.py",
+            "subprocess.run(\n"
+            '    "curl https://api.openai.com/v1/items",\n'
+            "    shell=True,\n"
+            ")\n",
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)